marklogic/tests/test_marklogic.py

```diff
@@ -121,6 +122,8 @@ def test_check_with_filters(aggregator):
     # Resource filter only
     for metric in STORAGE_HOST_METRICS + RESOURCE_STORAGE_FOREST_METRICS:
         aggregator.assert_metric_has_tag(metric, 'forest_name:Security', count=1)
+    for metric in RESOURCE_STATUS_DATABASE_METRICS:
+        aggregator.assert_metric_has_tag(metric, 'database_name:Documents', count=1)
```

Comment:
how was aggregator.assert_all_metrics_covered() passing before if RESOURCE_STATUS_DATABASE_METRICS were not asserted?

Reply:
Because this PR added a database include filter `{'resource_type': 'database', 'pattern': '^Doc'}` (https://github.com/DataDog/integrations-core/pull/7946/files#diff-3f72f4ba7139a9f13ef0760a5e4d3165554babbf149b88f1649b623c98bab023R51)
So metrics about the `Documents` database are now collected in the test environment